test/NuGet.Core.Tests/NuGet.Versioning.Test/VersionRangeTests.cs

```diff
@@ -146,6 +146,32 @@ public void VersionRange_MetadataIsIgnored_FormatRemovesMetadata()
             Assert.Equal("[1.0.0, 2.0.0]", bothMetadata.ToLegacyString());
         }
 
+        [Fact]
+        public void VersionRange_FloatAllStable_ReturnsCorrectPrints()
```

Comment:
Worth having a true round trip test? `new` -> `ToNormalizedString` -> `Parse` then compare all properties

Reply:
Good point. I thought there was one, but I couldn't find it. 

I'll add a few.

Reply:
Fixed.